fem/bilinearform.cpp

```diff
@@ -122,8 +122,7 @@ void BilinearForm::SetAssemblyLevel(AssemblyLevel assembly_level)
    switch (assembly)
    {
       case AssemblyLevel::FULL:
```

Comment:
Should calling `SetAssemblyLevel(AssemblyLevel::LEGACYFULL)` be supported?

Reply:
Done.